jsonthebuilder/src/components/ResumeContainer.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import sampleResume from "../assets/sampleResume";
+import ResumeField from "./ResumeField";
+
+export default function ResumeContainer() {
+  // const sample = {sampleResume: "peos"};
+  // const [information, setInformation] = useState(JSON.parse(sample))
+  const jasson = JSON.stringify(sampleResume);
+  const obj = [JSON.parse(jasson)];
```

Comment:
I don't think you actually need this step - you first stringify sampleResume, and then parse it back into an object, which I believe are essentially opposite methods. To use the object, I think you could've just used the `sampleResume` import.